packages/docusaurus-theme-classic/src/theme/Tabs/index.tsx

```diff
@@ -192,11 +107,7 @@ function TabsComponent(props: Props): JSX.Element {
 
     if (newTabValue !== selectedValue) {
       blockElementScrollPositionUntilNextRender(newTab);
-      setSelectedValue(newTabValue);
```

Comment:
I thought of moving `blockElementScrollPositionUntilNextRender` into `tabGroupChoice.tsx`, but decided to leave it as-is. Let me know if you think it's better to move it.